src/System.Device.Gpio/System/Device/Gpio/GpioChipException.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace System.Device.Gpio
+{
+    class GpioChipException : Exception
```

Comment:
>class GpioChipExceptio [](start = 4, length = 22)

This class would be public? or you want to keep it as private? usually we set the visibility specifier in front of every class we write.

Reply:
I think it shoudl be public, thanks